src/amo/routes.js

```diff
@@ -29,7 +29,10 @@ export default (
     <IndexRoute component={Home} />
     <Route path="addon/:slug/" component={DetailPage} />
     <Route path="addon/:addonSlug/reviews/" component={AddonReviewList} />
-    {/* Hack to make the proxy serve this URL from addons-server */}
+    {/* These user routes are to make the proxy serve each URL from */}
+    {/* addons-server until we can fix the :visibleAddonType route below. */}
+    {/* https://github.com/mozilla/addons-frontend/issues/2029 */}
+    <Route path="users/edit/" component={NotFound} />
```

Comment:
I wonder if this should have a placholder since there's quite a lot of options for other urls one could hit: https://github.com/mozilla/addons-server/blob/master/src/olympia/users/urls.py#L20